integration-tests/utils/Tokens.ts

```diff
@@ -0,0 +1,40 @@
+export class Tokens {
+
+    static removeBindingsAndTokens(){
+        cy.getCookie('cs_jwt').should('exist').then((cookie) => { 
+            let token = cookie.value;
+            let namespace = Cypress.env("USERNAME").toLowerCase();
+
+            removeAllResources(token, namespace, "spiaccesstokenbinding");
+            removeAllResources(token, namespace, "spiaccesstoken");
+        });
+    } 
+}
+
+function removeAllResources(token: string, namespace: string, resourceType : string) {
+    const getResources = {
+        method: 'GET',
+        url: `https://env-ephemeral-alqcwz-yjsfmfej.apps.c-rh-c-eph.8p0c.p1.openshiftapps.com/api/k8s/apis/appstudio.redhat.com/v1beta1/namespaces/${namespace}/${resourceType}s`,
+        headers: {
+            authorization: `Bearer ${token}`,
+            accept: 'application/json',
+        }
+    }
+
+    cy.request(getResources).its('body').then(respBody => {
+        for (const item of respBody.items) {
+            let resourceName = item.metadata.name;
+            const removeResources = {
+                method: 'DELETE',
+                url: `https://env-ephemeral-alqcwz-yjsfmfej.apps.c-rh-c-eph.8p0c.p1.openshiftapps.com/api/k8s/apis/appstudio.redhat.com/v1beta1/namespaces/${namespace}/${resourceType}s/${resourceName}`,
```

Comment:
same as above, might be better to extract the url to a variable so it's guaranteed consistency between the requests